sdk/release/rotation

```diff
@@ -1,11 +1,8 @@
 # If you are adding someone to the rotation, remember to also give them permission
 # to access an ad-hoc machine for Windows testing. This can be done by opening a PR
 # like the following: https://github.com/DACH-NY/daml-language-ad-hoc/pull/37
-U03SJDAU3BQ basvangijzel-DA
-U04M62HVADS samuel-williams-da
-U02DAFHFXUN moisesackerman-da
-U5LC1FB6D gerolf-da
```

Comment:
Should we be removing Gerolf, or was he on this list for other reasons?

Reply:
yes